src/op_mode/serial.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+#
+# Copyright (C) 2024 VyOS maintainers and contributors
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 or later as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import sys, typing
+
+import vyos.opmode
+from vyos.utils.serial import restart_login_consoles as _restart_login_consoles
+
+def restart_serial_tty(device_name: typing.Optional[str]):
```

Comment:
I think we should have the API match the CLI as closely as possible. This function will be automatically exposed in the GraphQL schema, so if the command is `restart serial console [device <dev>]`, the function should also be `restart_serial_console(device:  typing.Optional[str])`.

Reply:
What about “restart_console”?
Request will be RestartConsoleSerial

Reply:
Good point, that would be even closer to the CLI.